app/presenters/non_edition_result.rb

```diff
@@ -0,0 +1,60 @@
+class NonEditionResult < SearchResult
+  include ERB::Util
+
+  def to_hash
+    super.merge({
+      metadata: metadata,
+      metadata_any?: metadata.any?,
+    })
+  end
+
+  def format
+    result["document_type"]
+  end
+
+  def metadata
+    data = [
+      last_update,
+      display_type,
+      organisations,
+    ]
+
+    data.select(&:present?)
+  end
+
+private
+  def last_update
+    if result["last_update"]
+      result["last_update"].to_date.strftime("%e %B %Y")
+    end
+  end
+
+  def display_type
+    case format
+    when "aaib_report"
```

Comment:
I think this would be more readable as a hash lookup:

```ruby
  FORMAT_NAMES = {
    "aaib_report" => "AAIB report",
    "cma_case" => "CMA case",
  }

  FORMAT_NAMES.fetch(format, format.humanize)
```

(Or with a bit more logic if you want to avoid the call to humanize in the default case.)